src/main/java/seedu/address/logic/commands/EditRemarkCommand.java

```diff
@@ -0,0 +1,87 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.client.Client;
+import seedu.address.model.client.NameEqualsKeywordPredicate;
+import seedu.address.model.remark.Remark;
+import seedu.address.model.remark.UniqueRemarkList;
+
+
+/**
+ * Edits the details of an existing transaction in one client in the address book.
```

Comment:
`transaction` should be changed to `remark`.

Reply:
Changed thank you